sockeye/layers.py

```diff
@@ -46,11 +48,1542 @@ def activation(data: mx.sym.Symbol, act_type: str) -> mx.sym.Symbol:
         # Approximation of x * gaussian_cdf(x) used by Hendrycks and Gimpel
         return 0.5 * data * (1 + mx.sym.Activation((math.sqrt(2 / math.pi) * (data + (0.044715 * (data**3)))),
                                                    act_type="tanh"))
+    elif act_type == C.NO_ACTIVATION:
+        return data
     else:
         return mx.sym.Activation(data, act_type=act_type)
 
 
-class LayerNormalization:
+class Layer(ABC):
```

Comment:
docstring

Reply:
I'm not sure what type of doc string I could add that would add any value :P

Reply:
maybe you could add a short description of the type hierarchy that you are creating. `Layer` being the most basic class that is used for custom enc/dec sequences.